.github/workflows/shellcheck.yml

```diff
@@ -11,8 +13,6 @@ jobs:
     - name: Run ShellCheck
       uses: ludeeus/action-shellcheck@master
       env:
-        SHELLCHECK_OPTS: -e SC1091 -e SC2034 -e SC2154 -e SC3040 -e SC2148
+        SHELLCHECK_OPTS: -x
```

Comment:
Add "-x" so we can actually include files not specified on the command line. Remove exclusions.

Reply:
this didn't work when I tried it previously

Reply:
> this didn't work when I tried it previously

@brettin It does, the problem was the issue I marked with "This ☝️☝️☝️☝️☝️" further down in this PR (path to the included file needs to be relative to shellchecks CWD). The error message is very similar to when -x isn’t given, that’s what confused you (you can try locally for comparison)

When I looked at the review where you ran into these issues, I didn’t realize that the file was included from two places in two different directories (and the problem was caused by the second one using a relative path within the same subdirectory).